apps/autoscale-by-metric.html.markerb

```diff
@@ -0,0 +1,181 @@
+---
+title: Autoscale based on metrics
+layout: docs
+nav: firecracker
+---
+
+Most applications on Fly.io can use the request-based autoscaling that is built into our proxy by setting [the soft & hard limits in your fly.toml](/docs/reference/configuration/#http_service-concurrency). However, some applications need to scale on other metrics besides requests such as a work queue depth. In these instances, you can use our open source [fly-autoscaler](https://github.com/superfly/fly-autoscaler) to scale your workload.
+
+The metrics-based autoscaler works by collecting metrics from different sources, such as Prometheus or Temporal, and computing the number of required Machines based on those metrics. This reconciliation process happens on a loop every 15 seconds by default. We use the [Expr language](https://expr-lang.org/) for defining target Machine counts which gives a rich set of built-in arithmetic functions.
+
```

Comment:
@benbjohnson Do you think it would be useful to talk about create/destroy vs. stop/start here? Or down below in the Stop/start Machines section?
Something like:
```suggestion
You have the option of scaling to a target by starting and stopping a group of existing Machines, or by creating and destroying Machines. There are benefits and drawbacks to each of these options.

If you scale by starting and stopping existing Machines, then your Machines will start up very quickly. You will pay for the Machines's CPU and RAM when they're running and for the rootfs when they're stopped.

If you scale by creating and destroying Machines, then your Machines will be slightly slower to reach a `started` state since it takes longer to cerate a Machine than to start one. You won't need to create a "pool" of Machines. You will only pay for the Machines's CPU and RAM when they're running and won't need to pay for rootfs since the Machines are destroyed when not needed.
```